les/server.go

```diff
@@ -279,3 +296,18 @@ func (s *LesServer) capacityManagement() {
 		}
 	}
 }
+
+func (s *LesServer) getClient(id enode.ID) *clientPeer {
+	if node := s.ns.GetNode(id); node != nil {
+		if p, ok := s.ns.GetField(node, clientPeerField).(*clientPeer); ok {
+			return p
+		}
+	}
+	return nil
+}
+
+func (s *LesServer) dropClient(id enode.ID) {
```

Comment:
No, it is the server handler's job to do that. Disconnection is either initiated remotely or by us in the p2p layer, then the server handler will exit `handle` function and remove the NSM entry. This will notify everyone else about the disconnection.

Reply:
Yeah, I saw that...